client/src/modules/journal/modals/trialBalanceMain.body.html

```diff
@@ -28,6 +15,17 @@
       <span translate>FORM.BUTTONS.VIEW_ERRORS</span>
     </button>
   </div>
+
+  <div class="col-xs-6 text-right">
+    <button
+      id="export"
+      type="button"
```

Comment:
Good reference!

I've also run into the case where `<button>Cancel</button>` is interpreted as `<button type="submit">Cancel</button>` and submits the form instead of canceling it (if in a `<form>` element).  I'm not sure how often this occurs, but it is mighty annoying and confusing.